bin/kramdown-rfc2629

```diff
@@ -71,21 +71,30 @@ def xml_from_sections(input)
   open_refs = ps[:ref] || { }       # consumed
 
   norm_ref = { }
+  label_ref = { }
 
   # convenience replacement of {{-coap}} with {{I-D.ietf-core-coap}}
   # collect normative/informative tagging {{!RFC2119}} {{?RFC4711}}
   sechash.each do |k, v|
     next if k == "fluff"
-    v.gsub!(/{{(?:([?!])(-)?|(-))([\w.\-]+)}}/) do |match|
+    v.gsub!(/{{(?:([?!])(-)?|(-))([\w.\/_\-]+(?: ([\w._\-]+))?)}}/) do |match|
```

Comment:
Testing shows that this works, but this won't match on `{{? SST}}`.  I wonder if this should be `/{{(?:([?!])(-)?|(-))((?:[\w.\/_\-]+)?(?: ([\w._\-]+))?)}}/` instead.